src/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Style.css";
+import constant from "./constants";
+
+let colors = constant.colors;
```

Comment:
You can import this constant like this: `import { colors } from './constants';`.